src/Visualization.jl

```diff
@@ -17,13 +17,37 @@ function Makie.plot!(f::Makie.GridPosition, tn::TensorNetwork{A}; labels = false
     scene = Makie.Scene()
     default_attrs = Makie.default_theme(scene, GraphPlot)
 
+    transform!(tn, HyperindConverter)
```

Comment:
If you use `transform!`, you're modifying the `TensorNetwork` of the user. Just do a shallow copy of it (tensors are copied but their arrays not).
```suggestion
    # NOTE hyper edges cannot be easily drawn. transform them into COPY-tensors instead.
    tn = transform(tn, HyperindConverter)
```